balanced_vrp_clustering.gemspec

```
@@ -4,13 +4,8 @@ Gem::Specification.new do |s|
   s.date = '2020-08-05'
   s.summary = 'Gem to clusterize points of a given VRP.'
   s.authors = 'Mapotempo'
-  s.files = [
-    "lib/balanced_vrp_clustering.rb",
-    "lib/helpers/helper.rb",
-    "lib/helpers/hull.rb",
-    "lib/concerns/overloadable_functions.rb"
-  ]
-  s.require_paths = ["lib", "test"]
+  s.files = %w[lib/balanced_vrp_clustering.rb lib/helpers/helper.rb lib/helpers/hull.rb lib/concerns/overloadable_functions.rb]
```

Can we keep the previous style unless you have a specific reason? It is easier to see the content and the diff becomes  obvious when the content is modified.
```suggestion
  s.files = %w[
    lib/balanced_vrp_clustering.rb
    lib/helpers/helper.rb
    lib/helpers/hull.rb
    lib/concerns/overloadable_functions.rb
  ]
```

---

Yes, it's perfect ! ^^
I Commit suggestion